Header header
sensor_msgs/LaserScan data
